yarn.lock

```diff
@@ -3,3196 +3,3399 @@
 
 
 "@babel/code-frame@7.12.11":
-  version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
-  integrity sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw==
+  "integrity" "sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw=="
+  "resolved" "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz"
```

Comment:
I noticed that the "resolved" fields on all dependencies changed from pointing to registry.yarnpkg.com to registry.npmjs.org. I'm not sure if it makes sense to make that change. If you have a reason for this change, let me know and let's do it in a separate PR! 

For this PR, would you mind looking into reverting these changes?

Reply:
Absolutely! this was autogenerated, so I can definitely roll back.

Reply:
Done!